### PR TITLE
Doc on updating services

### PR DESCRIPTION
**Pull request by elisabettai**
## What do these changes do?
Added information on how to update services

## Related issue number
#608

## Checklist

- [X] I think the documentation is well written and up to date
- [X ] Do NOTE have very large media files (e.g. photos, videos, etc)
- [X] My spell-checker passes
- [ ] If you write a new section, i take ownership adding my user in .github/CODEOWNERS

### CODE REVIEW

File: docs/study_setup/add_service.md [hunk 1]
```diff
@@ -17,4 +17,20 @@ You can also access the Service Catalog by clicking on the input or output port
...
+
+You can also open the same window by clicking on the three dot button on the study card and then selecting the update icon from the left bar.
+
+![updateservice](https://user-images.githubusercontent.com/33152403/158360143-435f7db9-fcc4-4a7e-967d-1df7d6e9b0e5.gif)
+
+Some points to keep in mind:
+- It is only possible to update to the latest version
+- All settings and data will be preserved when updating
+- You can only update the patch version (e.g. from version 1.0.1 to 1.0.4 and not from 1.0.1 to 1.1.1 or 2.0.1)
+- To update to minor/major versions you have to open the study and replace the node with the new version. In some cases, a minor/major update can break the pipeline and you will need to adapt your study to work with the updated service.
```
Comment (KZzizzle) [inline docs/study_setup/add_service.md:36]: suggested change:
...replace the node with the new version... -> ...delete the outdated node and then create a new node with the latest Service version... + make sure to download the node data (if there is any) locally before deleting the original node.

**Review by KZzizzle — APPROVED** [review]
Very nice! Minor request - capitalize all instances of Study (just to be consistent with the rest of the docs)

File: docs/study_setup/add_service.md (hunk 1 above)
Comment (elisabettai) [reply docs/study_setup/add_service.md]: Much better like that, thanks!

**Comment by elisabettai** [comment]
> Very nice! Minor request - capitalize all instances of Study (just to be consistent with the rest of the docs)

Done (for the "Add Service, etc.. page"). The same for "Service". Will keep that in mind for future updates!

**Pull request merged**